packages/rocketchat-lib/server/functions/saveUser.js

```diff
@@ -167,7 +167,9 @@ RocketChat.saveUser = function(userId, userData) {
 			updateUser.$set.requirePasswordChange = userData.requirePasswordChange;
 		}
 
-		updateUser.$set['emails.0.verified'] = !!userData.verified;
+		if (userData.verified) {
+			updateUser.$set['emails.0.verified'] = true;
```

Comment:
Can't I set as not verified? Shouldn't be more correct to check if `userData.verified` is boolean and then allow to change, that way we do not require to pass that information but allow admins to revert some mistake.